tests/ci/common_posix_setup.sh

```diff
@@ -143,63 +143,31 @@ function build_and_run_minimal_test {
   run_cmake_custom_target 'run_minimal_tests'
 }
 
-function print_system_and_dependency_information {
-  if command -v cmake &> /dev/null
-  then
-    echo ""
-    echo "CMake version:"
-    cmake --version
-  fi
-  if command -v cmake3 &> /dev/null
-  then
-    echo ""
-    echo "CMake version (cmake3 executable):"
-    cmake3 --version
-  fi
-  if command -v go &> /dev/null
-  then
-    echo ""
-    echo "Go version:"
-    go version
-  fi
-  if command -v perl &> /dev/null
-  then
-    echo ""
-    echo "Perl version:"
-    perl --version
-  fi
-  # Ninja executable names are not uniform
-  if command -v ninja-build &> /dev/null
-  then
-    echo ""
-    echo "Ninja version"
-    ninja-build --version
-  elif command -v ninja &> /dev/null
-  then
-    echo ""
-    echo "Ninja version"
-    ninja --version
-  fi
-  if command -v gcc &> /dev/null
-  then
-    echo ""
-    echo "GCC version:"
-    gcc --version
-    echo ""
-    echo "G++ version:"
-    g++ --version
-  elif command -v clang &> /dev/null
-  then
-    echo ""
-    echo "Clang version:"
-    clang --version
-  fi
-  if command -v make &> /dev/null
+function print_executable_information {
+  EXE_NAME=${1}
+  EXE_ARGUMENT=${2}
+  LABEL=${3}
+
+  if command -v ${EXE_NAME} &> /dev/null
   then
     echo ""
-    echo "Make version:"
-    make --version
+    echo "${LABEL}:"
+    ${EXE_NAME} ${EXE_ARGUMENT}
   fi
```

Comment:
Should there be an `else` block to log the fact that the command is not found -- i.e., be explicit instead of implicit about missing tools?

Reply:
I guess that doesn't hurt